functional-tests/changed-ftest/src/test/java/org/arquillian/smart/testing/ftest/ITIntegration.java

```diff
@@ -0,0 +1,11 @@
+package org.arquillian.smart.testing.ftest;
+
+import org.junit.Test;
+
+public class ITIntegration {
```

Comment:
I'm totally st...d/blind or my `git diff` command doesn't work correctly - this shouldn't be part of this commit of course. I'll remove it in a sec.